examples/hosts/app-integration/external-controller/src/view.ts

```diff
@@ -6,29 +7,21 @@
 import { IKeyValueDataObject } from "./kvpair-dataobject";
 
 /**
- * Render an IDiceRoller into a given div as a text character, with a button to roll it.
+ * Render Dice into a given HTMLElement as a text character, with a button to roll it.
  * @param dataObject - The Data Object to be rendered
- * @param div - The div to render into
+ * @param div - The HTMLElement to render into
  */
-export function renderDiceRoller(DO: IKeyValueDataObject, div: HTMLDivElement) {
+export function renderDiceRoller(DO: IKeyValueDataObject) {
     const dataKey = "dataKey";
     // Set init value for dataKey
     DO.set(dataKey, 1);
-    const wrapperDiv = document.createElement("div");
-    wrapperDiv.style.textAlign = "center";
-    div.append(wrapperDiv);
 
-    const diceCharDiv = document.createElement("div");
-    diceCharDiv.style.fontSize = "200px";
+    const diceCharDiv = document.getElementById("dice")!;
```

Comment:
I'm actually not a fan of this :-/  It imposes an external dependency on the consumer of the view, basically reducing the encapsulation of the view.  It also prevents multiply-rendering the view (e.g. today you can render the dice twice, or render two different dice rollers by just calling this same render function twice).

Reply:
Let's have the dialogue here... It's a good question.

Reply:
I think if we're in the data + API layer, then we'll mostly have developers not encapsulate the view. Is that something we're *un*comfortable with?

Reply:
A view author using a view framework (React, Vue, etc.) will have encapsulated view components.  And even for vanillaJS views, if they are consuming our data + API layer to build a reusable component (e.g. drawing multiple poll widgets on a single page) then encapsulation and multiple-rendering is likely still valuable to them.  I think the view pattern is independent from their Fluid consumption pattern.

Reply:
true...it's not really a view anymore, it'd be a controller

Reply:
Yep. We should pick an agenda for this directory. If it's "Hello World", then I think moving DOM creation to the html is reasonable. That change simplifies things and isn't meant to be indicative of the whole app structure.

If this is an exploration of the surface, then we keep the vie controller coupled with the view to mimic the more common view library patterns.

Thoughts?